scripts/core/superdesk-api.d.ts

```diff
@@ -2160,6 +2160,8 @@ declare module 'superdesk-api' {
         previewComponent: React.ComponentType<IPreviewComponentProps>;
         configComponent?: React.ComponentType<IConfigComponentProps<IConfig>>;
         templateEditorComponent?: React.ComponentType<ITemplateEditorComponentProps<IConfig>>;
+
+        onTemplateCreate?(field: IConfig): any;
```

Comment:
https://github.com/superdesk/superdesk-client-core/pull/4033#discussion_r763888174

Can you also pass existing field value to the function?